include/diaspora.php

```diff
@@ -932,18 +932,20 @@ private static function post_allow($importer, $contact, $is_comment = false) {
 			logger("defining user ".$contact["nick"]." as friend");
 		}
 
-		// We don't seem to like that person
+		// Is this contact wanted?
```

Comment:
@Quix0r has changed the specific sentence with this PR.
My comment was more an answer to @annando 's comment to highlight that a comment should be more descriptive to give newcomers (or sometimes also main developers) the chance to understand (and learn) friendicas internal logic